src/components/Database/DatabasePicker.scss

```diff
@@ -0,0 +1,16 @@
+.Database-Picker {
```

Comment:
nit: missing license?

Reply:
Ugh somehow google/addlicense does not handle scss files so i added manually. Let me find another tool and also add a check later.